README.md

```diff
@@ -115,45 +109,69 @@ sudo make install
 The generic target allows running placement and routing for arbitrary custom architectures.
 
 ```
-cmake -DARCH=generic .
+cmake . -DARCH=generic
 make -j$(nproc)
 sudo make install
 ```
 
 An example of how to use the generic flow is in [generic/examples](generic/examples). See also the [Generic Architecture docs](docs/generic.md).
 
-Additional notes for building nextpnr
--------------------------------------
+### Multiple architectures
 
-Use cmake `-D` options to specify which version of nextpnr you want to build.
+To build nextpnr for multiple architectures at once, a semicolon-separated list can be used with `-DARCH`.
+
+```
+cmake . -DARCH="ice40;ecp5"
+make -j$(nproc)
+sudo make install
+```
 
-Use `-DARCH=...` to set the architecture. It is a semicolon separated list.
-Use `cmake . -DARCH=all` to build all supported architectures.
```

Comment:
I think `-DARCH=all` is worth mentioning.

Reply:
Added in the section on multiple architectures